contracts/cw20-ics20/src/ibc.rs

```diff
@@ -442,8 +441,7 @@ mod test {
                 channel_id: my_channel.to_string(),
             },
             sequence: 3,
-            timeout_block: None,
-            timeout_timestamp: Some(1665321069000000000u64),
+            timeout: IbcTimeout::with_timestamp(Timestamp::from_seconds(1665321069)),
```

Comment:
If you like it short:

```suggestion
            timeout: Timestamp::from_seconds(1665321069).into(),
```

Not sure this improves the code though